rmw_fastrtps_shared_cpp/src/rmw_request.cpp

```diff
@@ -38,16 +39,18 @@ __rmw_send_request(
   const void * ros_request,
   int64_t * sequence_id)
 {
-  assert(client);
-  assert(ros_request);
-  assert(sequence_id);
+  RMW_CHECK_ARGUMENT_FOR_NULL(client, RMW_RET_INVALID_ARGUMENT);
 
-  rmw_ret_t returnedValue = RMW_RET_ERROR;
+  RMW_CHECK_TYPE_IDENTIFIERS_MATCH(
+    client handle,
+    client->implementation_identifier, identifier,
+    return RMW_RET_INCORRECT_RMW_IMPLEMENTATION);
 
-  if (client->implementation_identifier != identifier) {
-    RMW_SET_ERROR_MSG("node handle not from this implementation");
-    return RMW_RET_ERROR;
-  }
+  RMW_CHECK_ARGUMENT_FOR_NULL(ros_request, RMW_RET_INVALID_ARGUMENT);
+
+  RMW_CHECK_ARGUMENT_FOR_NULL(sequence_id, RMW_RET_INVALID_ARGUMENT);
+
+  rmw_ret_t returnedValue = RMW_RET_ERROR;
```

Comment:
@Lobotuerk nit: consider removing all unnecessary blank lines above.

Reply:
@hidmic 8f130e5 like this?